go/windows/web.config

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <system.webServer>
+        <rewrite>
+            <rules>
+                <clear />
+
+                <!-- Links for Keyman for Windows 14.0 onward -->
+
+                <rule name="/go/windows/X.Y/download-keyboards" stopProcessing="true">
+                  <match url="^([1-9][0-9]\.[0-9])/download-keyboards" />
+                  <action type="Redirect" url="/keyboards?embed=windows" />
+                </rule>
+
+                <rule name="/go/windows/X.Y/keep-in-touch" stopProcessing="true">
+                  <match url="^([1-9][0-9]\.[0-9])/keep-in-touch" />
+                  <action type="Redirect" url="/desktop/keepintouch-100" /> <!-- TODO: move to /windows/keepintouch-140 -->
+                </rule>
+
+                <rule name="/go/windows/X.Y/issue-1285" stopProcessing="true">
+                  <match url="^([1-9][0-9]\.[0-9])/issue-1285" />
+                  <action type="Redirect" url="https://blog.keyman.com/2018/11/keyman-windows-10-1803-and-amharic-tigrinya-sinhala" />
+                </rule>
+
+                <!-- TODO: i18n needs to rewrite this to our home page for crowdin -->
+                <rule name="/go/windows/X.Y/create-locale" stopProcessing="true">
+                  <match url="^([1-9][0-9]\.[0-9])/create-locale" />
+                  <action type="Redirect" url="https://secure.tavultesoft.com/keyman/support/locale/" />
```

Comment:
Is it correct that this redirects to tavultesoft.com?

Reply:
Note the TODO comment for when we come to i18n. We have nowhere else ready to redirect to at present.